data/repos.yml

```diff
@@ -117,6 +134,29 @@
   dashboard_url: https://grafana-paas.cloudapps.digital/d/xonj40imk/data-gov-uk?refresh=1m&orgId=1
   production_url: https://ckan.publishing.service.gov.uk/
 
+# ditto ^ https://github.com/alphagov/ckanext-datagovuk/blob/4ae4e0eb03c88b4526672527490a6546e95c4d78/docker/ckan/Dockerfile#L105-L112
+- repo_name: ckanext-dcat
+  type: data.gov.uk apps
+  team: "#govuk-datagovuk"
+  sentry_url: false
+  dashboard_url: false
+- repo_name: ckanext-harvest
+  type: data.gov.uk apps
+  team: "#govuk-datagovuk"
+  sentry_url: false
+  dashboard_url: false
+- repo_name: ckanext-spatial
+  type: data.gov.uk apps
+  team: "#govuk-datagovuk"
+  sentry_url: false
+  dashboard_url: false
+
+- repo_name: ckanext-spatial
+  # Propose archiving. Not touched since 2018, not obviously referenced anywhere.
+
+- repo_name: col-prototypes
+  # Propose that the Cost of Living team add this (very new!) repo in a separate PR.
```

Comment:
CC @Rosa-Fox / @MartinJJones 🙏

Reply:
It does look like it should be on a separate PR, but I've not been involved in the setup of this repo or had to use it myself. I have messaged Jon Greer with a link to this section.

Reply:
Thanks @MartinJJones - any update from Jon?

Reply:
@MartinJJones any update from Jon? I see there was last activity on March 7th. I suspect the repo can be archived. I'm going to request archival in the Tech Leads channel unless I hear otherwise.

Reply:
Sorry for the late reply!, the last update I had from Jon was that the repo is used to test and demo designs. He was looking at creating a PR to add the prototype to the developers docs, something like:

```YML
- repo_name: col-prototypes
  type: Prototype
  team: “#cost-of-living-team”
  management_url: https://dashboard.heroku.com/apps/col-prototype
  production_hosted_on: heroku
```

But considering the last activity was on March 7th, and also that "Prototype" would be a new application type, I think it makes sense to archive it instead. I'll drop Jon a message as well to this thread to let him know.